### PR TITLE
PWGHF: Add raw prompt fraction to compute_fraction_cutvar.py output

### PR DESCRIPTION
**Pull request by fchinu**
This PR adds the raw prompt fraction to the output of the compute_fraction_cutvar.py script.

I have changed `np.float` to `float` as the former is now [deprecated](https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations)

### CODE REVIEW

File: PWGHF/D2H/Macros/compute_fraction_cutvar.py [hunk 1]
```diff
@@ -78,6 +94,19 @@ def main(config):
...
+        fillstyle=0,
+        markerstyle=ROOT.kFullSquare,
+    )
+
+    set_object_style(
+        hist_frac_raw_nonprompt,
+        color=ROOT.kAzure + 4,
+        fillstyle=0,
+        markerstyle=ROOT.kFullSquare,
+    )
```
Comment (fgrosa) [inline PWGHF/D2H/Macros/compute_fraction_cutvar.py:109]:
```suggestion
        if cfg["central_efficiency"]["computerawfrac"]:
            hist_frac_raw_prompt = hist_rawy[0].Clone("hRawFracPrompt")
            hist_frac_raw_nonprompt = hist_rawy[0].Clone("hRawFracNonPrompt")
            hist_frac_raw_prompt.GetYaxis().SetTitle("raw fraction prompt")
            hist_frac_raw_nonprompt.GetYaxis().SetTitle("raw fraction non-prompt")
            set_object_style(
            hist_frac_raw_prompt,
            color=ROOT.kRed + 1,
            fillstyle=0,
            markerstyle=ROOT.kFullSquare,
        )

        set_object_style(
            hist_frac_raw_nonprompt,
            color=ROOT.kAzure + 4,
            fillstyle=0,
            markerstyle=ROOT.kFullSquare,
        )
```

File: PWGHF/D2H/Macros/compute_fraction_cutvar.py [hunk 2]
```diff
@@ -105,10 +134,20 @@ def main(config):
         hist_covariance.SetBinError(ipt + 1, 0)
         corr_frac_prompt = minimiser.get_corr_prompt_fraction()
         corr_frac_nonprompt = minimiser.get_corr_nonprompt_fraction()
+        raw_frac_prompt = minimiser.get_raw_prompt_fraction(
+            hist_central_effp.GetBinContent(ipt + 1), hist_central_effnp.GetBinContent(ipt + 1)
+        )
+        raw_frac_nonprompt = minimiser.get_raw_nonprompt_fraction(
+            hist_central_effp.GetBinContent(ipt + 1), hist_central_effnp.GetBinContent(ipt + 1)
+        )
```
Comment (fgrosa) [inline PWGHF/D2H/Macros/compute_fraction_cutvar.py:142]:
```suggestion
        if cfg["central_efficiency"]["computerawfrac"]:
            raw_frac_prompt = minimiser.get_raw_prompt_fraction(
                hist_central_effp.GetBinContent(ipt + 1), hist_central_effnp.GetBinContent(ipt + 1)
            )
            raw_frac_nonprompt = minimiser.get_raw_nonprompt_fraction(
                hist_central_effp.GetBinContent(ipt + 1), hist_central_effnp.GetBinContent(ipt + 1)
            )
```

File: PWGHF/D2H/Macros/compute_fraction_cutvar.py [hunk 3]
```diff
@@ -105,10 +134,20 @@ def main(config):
...
         hist_corrfrac_prompt.SetBinContent(ipt + 1, corr_frac_prompt[0])
         hist_corrfrac_prompt.SetBinError(ipt + 1, corr_frac_prompt[1])
         hist_corrfrac_nonprompt.SetBinContent(ipt + 1, corr_frac_nonprompt[0])
         hist_corrfrac_nonprompt.SetBinError(ipt + 1, corr_frac_nonprompt[1])
+        hist_frac_raw_prompt.SetBinContent(ipt + 1, raw_frac_prompt[0])
+        hist_frac_raw_prompt.SetBinError(ipt + 1, raw_frac_prompt[1])
+        hist_frac_raw_nonprompt.SetBinContent(ipt + 1, raw_frac_nonprompt[0])
+        hist_frac_raw_nonprompt.SetBinError(ipt + 1, raw_frac_nonprompt[1])
```
Comment (fgrosa) [inline PWGHF/D2H/Macros/compute_fraction_cutvar.py:150]:
```suggestion
        if cfg["central_efficiency"]["computerawfrac"]:
            hist_frac_raw_prompt.SetBinContent(ipt + 1, raw_frac_prompt[0])
            hist_frac_raw_prompt.SetBinError(ipt + 1, raw_frac_prompt[1])
            hist_frac_raw_nonprompt.SetBinContent(ipt + 1, raw_frac_nonprompt[0])
            hist_frac_raw_nonprompt.SetBinError(ipt + 1, raw_frac_nonprompt[1])
```

File: PWGHF/D2H/Macros/compute_fraction_cutvar.py [hunk 4]
```diff
@@ -158,6 +197,8 @@ def main(config):
     hist_covariance.Write()
     hist_corrfrac_prompt.Write()
     hist_corrfrac_nonprompt.Write()
+    hist_frac_raw_prompt.Write()
+    hist_frac_raw_nonprompt.Write()
```
Comment (fgrosa) [inline PWGHF/D2H/Macros/compute_fraction_cutvar.py:201]:
```suggestion
    if cfg["central_efficiency"]["computerawfrac"]:
        hist_frac_raw_prompt.Write()
        hist_frac_raw_nonprompt.Write()
```

**Review by fgrosa — CHANGES_REQUESTED** [review]
Hi @fchinu thanks a lot!
Apart from a couple of comments on the style, I would put this addition optional (see my suggestions below)

File: PWGHF/D2H/Macros/compute_fraction_cutvar.py [hunk 5]
```diff
@@ -157,7 +200,9 @@ def main(config):
     hist_corry_nonprompt.Write()
     hist_covariance.Write()
     hist_corrfrac_prompt.Write()
-    hist_corrfrac_nonprompt.Write()
```
Comment (fgrosa) [inline PWGHF/D2H/Macros/compute_fraction_cutvar.py:160]: I think that this has been deleted by mistake

Comment (fchinu) [reply PWGHF/D2H/Macros/compute_fraction_cutvar.py:160]: Hi @fgrosa, many thanks for spotting it! I have fixed it now

**Review by fgrosa — APPROVED** [review]

**Pull request merged**